README.md

```diff
@@ -1,4 +1,141 @@
-# Hapi.js Socket.io Redis Chat Example {Work in Progress}
+# Hapi.js Socket.io Redis Riot.js Chat Example [Work in Progress]
+
+This example adds [Riot.js](http://muut.com/riotjs/) to our popular [Hapi.js Socket.io and Redis Publish/Subscribe chat application example](https://github.com/dwyl/hapi-socketio-redis-chat-example), as per issue [#20]( https://github.com/dwyl/hapi-socketio-redis-chat-example/issues/20).
+
+The commits have been added in step-by-step so it should be simple to follow along. Watch this space!
+
++ [ ] Add links to riot documentation throughout & encourage pre-reading
```

Comment:
riot --> Riot (keeping consistent with other JS frameworks mentioned in the README)